codecs/webp_dec/example.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<script src='webp_dec.js'></script>
+<script>
+  const Module = webp_dec();
+
+  async function loadFile(src) {
+    const resp = await fetch(src);
+    return await resp.arrayBuffer();
```

Comment:
Just `return` here is fine. The `await` is redundant.

Reply:
Oh cool. I thought it would make the stack trace look different, but it doesn’t. TIL.

Reply:
https://jakearchibald.com/2017/await-vs-return-vs-return-await/ 😄